### PR TITLE
OSDOCS-3533: Added release note for ADFS as supported OIDC provider

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3533

Link to docs preview:
https://deploy-preview-45264--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes.html#ocp-4-11-auth-oidc-providers

Additional information:

### CODE REVIEW

**Review by jldohmann — APPROVED** [review]
LGTM!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -139,6 +139,18 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
...
 === Authentication and authorization
 
+[id="ocp-4-11-auth-oidc-providers"]
+==== Active Directory Federation Services is now a supported OIDC provider
+
+The Active Directory Federation Services OpenID Connect (OIDC) provider is now tested and supported with {product-title}.
+
+[NOTE]
+====
+Currently, it is not supported to use Active Directory Federation Services with {product-title} when custom claims are used.
```
Comment (xingxingxia) [inline release_notes/ocp-4-11-release-notes.adoc:149]: To be more elaborate, would it be better to link https://issues.redhat.com/browse/RFE-2770 (it is public) and/or https://bugzilla.redhat.com/show_bug.cgi?id=2072927?

Comment (bergerhoffer) [reply release_notes/ocp-4-11-release-notes.adoc:149]: @xingxingxia Normally I wouldn't mind linking to a BZ or Jira, but it looks like those have both been closed/rejected. So I don't know if those would be good to link to. It sounds like we're not going to change anything on our end for this, but instead hope that Microsoft will. wdyt?

Comment (xingxingxia) [reply release_notes/ocp-4-11-release-notes.adoc:149]: Yes, their closed/rejected statuses make it awkward to link them, though they indeed include elaborate info which may be useful. Anyway, let's not link then. Thx

**Pull request merged**